modules/nw-egressnetworkpolicy-object.adoc

```diff
@@ -143,6 +145,31 @@ spec:
       protocol: TCP
     - port: 443
 ----
+
+[id="configuringNodeSelector-example_{context}"]
+== Example nodeSelector for {kind}
+
+As a cluster administrator, you can allow or deny egress traffic to nodes in your cluster by specifying a label using `nodeSelector`. Labels can be applied to one or more nodes. The following is an example with the `type=user-node, region=east` labels:
```

Comment:
Below example is one key/value pair, better to remove "type=user-node" here to keep consistence

Reply:
Hi @huiran0826 I took that formatting from [here](https://docs.openshift.com/container-platform/4.12/nodes/scheduling/nodes-scheduler-node-selectors.html#nodes-scheduler-node-selectors-about_nodes-scheduler-node-selectors) to keep our docs consistent. With respect to constancy, can you please confirm whether this change is technically necessary or whether we should keep it as is?

Reply:
In my opinion, better to keep  the statement here consistent with your example.   The statement is "The following is an example with the `type=user-node, region=east` labels:" ,  but example is only for label "region=east".